layout/contact.layout.html

```diff
@@ -2,13 +2,13 @@
 	<h3>Pr&eacute;sidence</h3>
 	<p>Question sur notre association, notre programme ou nos activit&eacute;s.</p>
 	<!-- <p class="align_droite"><a href="mailto:contact@cbseraing.be">cbseraing@gmail.com</a></p> -->
-	<p class="align_droite"><a href="mailto:contact@cbseraing.be">contact@cbseraing.be</a></p>
+	<p class="align_droite"><a href={{HREF_EMAIL_PRESI}}>{{EMAIL_PRESI}}</a></p>
```

Comment:
There are missing double-quotes before `{{HREF...}}`, it needs to be: `<a href="{{HREF_EMAIL_PRESI}}">`